Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue13417.xaml

```diff
@@ -0,0 +1,92 @@
+        <StackLayout
+            Padding="12">
+            <Label
+                Text="Option 1"/>
+            <Frame>
+                <Frame.Style>
+                    <Style TargetType="Frame">
+                        <Setter Property="BorderColor" Value="Black" />
+                        <Setter Property="CornerRadius" Value="12" />
+                        <Setter Property="HeightRequest" Value="120" />
+                        <Setter Property="Background" Value="{StaticResource IssueLinearGradientBrush}"/>
+                        <Style.Triggers>
+                            <DataTrigger TargetType="Frame" Binding="{Binding Changed1}" Value="True">
+                                <Setter Property="Background" Value="Red"/>
+                            </DataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </Frame.Style>
+            </Frame>
+            <Button
+                Text="Update the Background (Option 1)"
+                Command="{Binding ChangeBackground1Command}"/>
+            <Label
+                Text="Option 2"/>
+            <Frame>
+                <Frame.Style>
+                    <Style TargetType="Frame">
+                        <Setter Property="BorderColor" Value="Black" />
+                        <Setter Property="CornerRadius" Value="12" />
+                        <Setter Property="HeightRequest" Value="120" />
+                        <Setter Property="Background" Value="{StaticResource IssueLinearGradientBrush}"/>
+                        <Style.Triggers>
+                            <DataTrigger TargetType="Frame" Binding="{Binding Changed2}" Value="True">
+                                <Setter Property="Background">
+                                    <SolidColorBrush Color="Red" />
+                                </Setter>
+                            </DataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </Frame.Style>
+            </Frame>
+            <Button
+                Text="Update the Background (Option 2)"
+                Command="{Binding ChangeBackground2Command}"/>
+            <Label
+                Text="Option 3"/>
+            <Frame>
+                <Frame.Style>
+                    <Style TargetType="Frame">
+                        <Setter Property="BorderColor" Value="Black" />
+                        <Setter Property="CornerRadius" Value="12" />
+                        <Setter Property="HeightRequest" Value="120" />
+                        <Setter Property="Background" Value="{StaticResource IssueLinearGradientBrush}"/>
+                        <Style.Triggers>
+                            <DataTrigger TargetType="Frame" Binding="{Binding Changed3}" Value="True">
+                                <Setter Property="BackgroundColor" Value="Red" />
+                                <Setter Property="Background" Value="{x:Null}" />
```

Comment:
If we have both the `BackgroundColor` (Color) and `Background` (Brush) properties assigned to a View, Background prevails. For that reason, in this case, in order to have it working as expected, a null value must be set to Background.
Should we change the behavior? What would be a better option?